### PR TITLE
chore(reports): remove break_even and cost_center

### PR DESCRIPTION
**Pull request by jniles**
This commit removes unused break_even and cost_center reports.

Partially addresses #6621.

### CODE REVIEW

**Comment by jniles** [comment]
@jmcameron this is ready for a review.

File: server/models/bhima.sql [hunk 1]
```diff
@@ -96,19 +96,12 @@ INSERT INTO unit VALUES
...
-  -- (221,'Update Distributions','TREE.UPDATE_DISTRIBUTION','',218,'/allocation_center/update'),
-  -- (222,'Fee Center Report','TREE.COST_CENTER_REPORT','Fee Center Report',286,'/reports/cost_center'),
-  -- (223,'Distribution keys','TREE.DISTRIBUTION_KEYS_MANAGEMENT','Distribution keys',218,'/allocation_center/allocation_key'),
+  (218,'Cost Center Management','TREE.COST_CENTER_MANAGEMENT','',0,'/cost_center'),
+  (219,'Cost Center Management','TREE.COST_CENTER','',218,'/cost_center'),
   (226,'Account Reference Type','TREE.ACCOUNT_REFERENCE_TYPE','Account Reference Type',1,'/account_reference_type'),
   (227,'Ward Module','TREE.WARD','Ward folder',0,'/WARD_FOLDER'),
   (228,'Ward Configurations','TREE.WARD_CONFIGURATION','Ward configuration module',227,'/ward/configuration'),
   (229,'Visits Registry','TREE.VISITS_REGISTRY','Visits registry',12,'/patients/visits'),
-  -- (230,'Break Even Reference','TREE.BREAK_EVEN_REFERENCE','Break Even Reference',1,'/break_even_reference'),
```
Comment (jmcameron) [inline server/models/bhima.sql:109]: 👍

File: server/models/bhima.sql [hunk 2]
```diff
@@ -154,7 +147,7 @@ INSERT INTO unit VALUES
   (283,'HR Reports','TREE.REPORTS','reports for the HR/Payroll modules',57,'/PAYROLL_FOLDER/reports'),
   (284,'Data Kit Reports','TREE.REPORTS','reports for the data collection modules',254,'/data_collection/reports'),
   (285,'Purchase Reports','TREE.REPORTS','reports for the purchasing modules',154,'/PURCHASE_FOLDER/reports'),
-  (286,'Fee Center Reports','TREE.REPORTS','reports for the cost center modules',218,'/cost_center/reports'),
+  (286,'Cost Center Reports','TREE.REPORTS','reports for the cost center modules',218,'/cost_center/reports'),
```
Comment (jmcameron) [inline server/models/bhima.sql:150]: 👍

File: server/models/bhima.sql [hunk 3]
```diff
@@ -203,60 +196,60 @@ INSERT IGNORE INTO `account_type` VALUES
...
+  ('stock_consumption_graph_report', 'REPORT.STOCK_CONSUMPTION_GRAPH_REPORT.TITLE'),
+  ('stock_entry', 'REPORT.STOCK.ENTRY_REPORT'),
+  ('stock_exit', 'REPORT.STOCK.EXIT_REPORT'),
+  ('stock_expiration_report', 'REPORT.STOCK_EXPIRATION_REPORT.TITLE'),
+  ('stock_movement_report', 'REPORT.STOCK_MOVEMENT_REPORT.TITLE'),
+  ('stock_sheet', 'REPORT.STOCK.INVENTORY_REPORT'),
+  ('stock_value', 'TREE.STOCK_VALUE'),
+  ('system_usage_stat', 'REPORT.SYSTEM_USAGE_STAT.TITLE'),
+  ('unpaid_invoice_payments', 'REPORT.UNPAID_INVOICE_PAYMENTS_REPORT.TITLE'),
+  ('visit_report', 'PATIENT_RECORDS.REPORT.VISITS');
```
Comment (jmcameron) [inline server/models/bhima.sql:252]: Thanks for alphabetizing!
Hard to check correspondence now but easier in the future.

**Review by jmcameron — APPROVED** [review]
Code changes look good.

Tested the changes with bhima_test.  Since these reports had already been disabled, there was not much to check.    I did try the current cost center pages and they seem to work, so nothing being used was deleted.

I did not try a migration.

**Pull request merged**